modules/kmm-example-module-cr.adoc

```diff
@@ -74,7 +74,7 @@ spec:
 ----
 <1> Required.
 <2> Optional.
-<3> Optional: Copies `/firmware/*` into `/var/lib/firmware/` on the node.
+<3> Optional: Optional: Copies the contents of this path into the path specified in `worker.setFirmwareClassPath` (which is preset to `/var/lib/firmware`) of the `kmm-operator-manager-config` configMap. This action occurs before `modprobe` is called to insert the kernel module.
```

Comment:
* "Optional:" dupe.
* Either "config map" or "`ConfigMap` object" per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting).
```suggestion
<3> Optional: Copies the contents of this path into the path specified in `worker.setFirmwareClassPath` (which is preset to `/var/lib/firmware`) of the `kmm-operator-manager-config` config map. This action occurs before `modprobe` is called to insert the kernel module.
```

Reply:
@adellape Ugh! How did I miss that? Thanks Good catch! Fixed.